pkg/crc/cluster/cluster.go

```diff
@@ -510,3 +510,29 @@ func DeleteMCOLeaderLease(ctx context.Context, ocConfig oc.Config) error {
 	_, _, err := ocConfig.RunOcCommand("delete", "-A", "lease", "--all")
 	return err
 }
+
+func CheckCorePodsRunning(ctx context.Context, ocConfig oc.Config) error {
```

Comment:
Maybe this could start with ` if err := WaitForOpenshiftResource(ctx, ocConfig, "pod"); err != nil {` ?